lib/decidim/questions/component.rb

```diff
@@ -163,6 +163,33 @@
       global = nil
     end
 
+    email = "committee-user-#{participatory_space.underscored_name}-#{participatory_space.id}@example.org"
+    name = "#{Faker::Name.name} #{participatory_space.id} #{n} committee"
```

Comment:
What does the "n" stand for?
`name = "#{Faker::Name.name} #{participatory_space.id} #{n} committee"`

Reply:
It was a mistake, sorry!